src/main/java/org/myrobotlab/framework/Service.java

```diff
@@ -156,6 +157,17 @@
   
   protected String serviceVersion = null;
   
+    /**
+   * map of keys to localizations - 
+   * <pre>
+   *  Match Service with current Locale of the Runtime service
+   *  Match Runtime with current Locale of the Runtime service.
+   *  Match Runtime with Default (english) Locale

```

Comment:
I think there should be a check for "Match Service with Default (english) Locale"  prior to falling back to runtime.

Reply:
done ..

Reply:
Javadoc wasn't updated...

Reply:
updated